features/cellular/TESTS/api/cellular_device/main.cpp

```diff
@@ -93,7 +93,7 @@ static void other_methods()
     TEST_ASSERT(device->get_queue() != NULL);
     TEST_ASSERT(device->hard_power_on() == NSAPI_ERROR_OK);
     TEST_ASSERT(device->soft_power_on() == NSAPI_ERROR_OK);
-    wait(5);
+    wait(10);
```

Comment:
@mudassar-ublox  It will impact all targets so better to use macro if it is for for SARA-U201. 
 
Please add @RobMeades in review too

Reply:
Integration manual states that _"The duration of SARA-G3 and SARA-U2 series modules’ switch-on routine can vary depending on the application / network settings and the concurrent module activities."_

I can't find specific time range

Reply:
Yes, I haven't find specific time in manual but as usually U201 took 12-13 sec to power up.

Reply:
Updated, Please review.